js/hitbtc.js

```diff
@@ -726,7 +726,7 @@ module.exports = class hitbtc extends Exchange {
     async fetchOrder (id, symbol = undefined, params = {}) {
         await this.loadMarkets ();
         let response = await this.tradingGetOrder (this.extend ({
-            'client_order_id': id,
+            'clientOrderId': id,
```

Comment:
Hey again! Thanks for your help!
Are you sure this edit is correct? According to HitBTC v1 reference that parameter should be `'client_order_id'` with underscores... from here: https://github.com/hitbtc-com/hitbtc-api/blob/master/APIv1.md
This is a v1 implementation, their current v2 is in `hitbtc2.js`...
Let me know if you still insist on this change, please. Thx!

Reply:
Hello,
Yes their repo is horrible ! As you can see all other endpoint use clientOrderId. 
I have double check it is not working with the underscore way

Reply:
Thanks! Just checked it on my side as well, indeed they don't accept client_order_id...